tests/keras/layers/convolutional_test.py

```diff
@@ -399,6 +399,12 @@ def test_averagepooling_2d():
                        'pool_size': (2, 2),
                        'data_format': 'channels_first'},
                input_shape=(3, 4, 5, 6))
+    layer_test(convolutional.AveragePooling2D,
+               kwargs={'strides': None,
+                       'padding': 'valid',
+                       'pool_size': (2, 2),
```

Comment:
?

Reply:
sorry, that was a stupid mistake .. my intent was to test input of None on pool_size parameter, not anything to do with strides. I can't run the tests on my current box and relying on the Jenkins feedback